tka/scenario_test.go

```diff
@@ -204,7 +204,7 @@ func TestNormalPropagation(t *testing.T) {
     `)
 	control := s.mkNode("control")
 
-	// Lets say theres a node with some updates!
+	// Let's say there's a node with some updates!
```

Comment:
unnecessary exclamation marks for something that's not actually exciting is another one of my pet peeves :)